discovery-provider/alembic/trigger_sql/handle_supporter_rank_ups.sql

```diff
@@ -0,0 +1,44 @@
+create or replace function handle_supporter_rank_up() returns trigger as $$
+declare
+  user_bank_tx user_bank_txs%ROWTYPE;
+begin
+  select * into user_bank_tx from user_bank_txs where user_bank_txs.slot = new.slot limit 1;
+
+  if user_bank_tx is not null then
+    -- create a notification for the sender and receiver
+    insert into notification
+      (slot, user_ids, timestamp, type, specifier, data)
+    values
+      (
+        new.slot,
+        ARRAY [new.sender_user_id],
```

Comment:
not too familiar with the code that uses `notification` table... _but_ if `user_ids` is an array why does this create two separate rows instead of doing?

```
[new.sender_user_id, new.receiver_user_id]
```

Reply:
yes, I kinda forget - I added the user_ids as an array for the subscription/fan out notifications and forgot

Reply:
hmm, actually I want to leave these as separate notifications because the way they are processed are different from sender and receiver